change/@fluentui-react-tree-12dac24a-04f5-472b-b3b6-d1fd3d82c7b4.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31779&commitId=87082f344f13ef6eb05c33a972afd537d0252919&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|2| Changed|
|Avatar Converged.size+inactive+badge.normal.chromium.png|0| Added|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.normal.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.normal.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>